packages/documentation/src/pages/platform/site-structure/client-overview.mdx

```diff
@@ -9,17 +9,19 @@ Welcome to the VA digital services platform front-end overview. The goal of this
+The build process has two primary kinds of functionality that it processes: static content pages and Javascript applications. Most of VA.gov consists of content that lives in Markdown files and the Drupal CMS, and is converted into HTML by a Metalsmith plugin. There are shared templates and other enhancements to make this content easier to structure.
 
-The build process has two primary kinds of functionality that it processes: static content pages and Javascript applications. Most of VA.gov consists of content that lives in Markdown files and is converted into HTML by a Metalsmith plugin. There are shared templates and other enhancements to make this content easier to structure. This content lives in `content/pages`, which also provides the basic URL structure of the site.
+The content build process for this is controlled by a tool called [Metalsmith](http://www.metalsmith.io/), which is a pluggable Node.js static site generator. In general, everything in Metalsmith is a plugin, so if you're looking for specific parts of the build process, expect to check out a plugin or two. We use Metalsmith because it's flexible and because it's Javascript based, it's easy to integrate with the rest of our tools.
 
 ## Javascript applications
 
-Javascript applications are the other type of functionality in vets-website. These applications are written in [React](https://reactjs.org/) and [Redux](https://redux.js.org/) and live in `src/applications/` in `vets-website`. For each app you'll find a `manifest.json` file that contains application information. You'll find the root url of the application, which you can use to navigate to that application on the site. These applications also usually have client-side routes, so opening that root url may allow you to click to other pages that are fully client-side, and not based on pages in `content/pages`.
+Javascript applications are the other type of functionality in vets-website. These applications are written in [React](https://reactjs.org/) and [Redux](https://redux.js.org/), and live in `src/applications/` in `vets-website`. For each app, you'll find a `manifest.json` file that contains application information as well as an entry in the `src/applications/registry.json`.  The `manifest.json` is used by Webpack in the application build, while the `registry.json` is used by the content build. See [Creating a new application](/veteran-facing-services-tools/getting-started/common-tasks/new-application/#manual-set-up) for more information on the difference between the `manifest.json` files and the `registry.json` file.
```

Comment:
Added a reference to the `registry.json`. It's just kind of hard to tell with this diff.